pkg/cmd/server/api/validation/master.go

```diff
@@ -192,5 +192,12 @@ func ValidatePolicyConfig(config api.PolicyConfig) fielderrors.ValidationErrorLi
 func ValidateProjectRequestConfig(config api.ProjectRequestConfig) fielderrors.ValidationErrorList {
 	allErrs := fielderrors.ValidationErrorList{}
 
+	if len(config.ProjectRequestTemplate) > 0 {
```

Comment:
> If it is optional, then shouldn't you use 'omitempty' in the json struct tag?

We've decided not to do that for any config options.  That way its easy to see all the different knobs that  you can turn.